spec/acceptance/apt_key_provider_spec.rb

```diff
@@ -263,16 +263,15 @@ def self.it_returns_a_failure_for(server)
         }
         EOS
 
-          apply_manifest(pp, expect_failures: true) do |r|
+          execute_manifest(pp, expect_failures: true) do |r|
             expect(r.stderr).to match(%r{could not resolve})
           end
         end
       end
 
       context 'ftp://' do
         before(:each) do
-          shell_ex("apt-key del #{CENTOS_GPG_KEY_SHORT_ID}",
-                   acceptable_exit_codes: [0, 1, 2])
```

Comment:
As far as I can see, ( https://github.com/puppetlabs/beaker-testmode_switcher/blob/master/lib/beaker/testmode_switcher/runner_base.rb ) beaker-testmode_switcher does not have acceptable_exit_codes and does not have an exit code handler matching [0,1,2]

It might be useful to update beaker-testmode_switcher to have an option that allows for exit code in [0,1,2] or to support acceptable_exit_codes

Reply:
@DavidS I tested that the return code here doesn't matter - I changed the command to `"apt-key del #{CENTOS_GPG_KEY_SHORT_ID}" || false` so it returns 1, and the test passes OK.